EUROTHRM/EUROTHRM-IOC-01App/Db/devEurothrm.db

```diff
@@ -224,6 +229,83 @@ record(ai, "$(P)OUT_MAX")
   }
 
 ###########
+# Cache & disable ramp when sensor disconneccted
+record(fanout, "$(P)SENSOR_CHECK")
+{
+   field(LNK1, "$(P)SENSOR_CONNECTED")
+   field(LNK2, "$(P)SENSOR_DISCONNECTED")
+}
```

Comment:
Can you get rid of this record if you set the `INPA` field in `SENSOR_(DIS)CONNECTED` to be `CP`?